sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -316,33 +305,14 @@ class Explorer extends Editor<ExplorerProps, ExplorerState> {
         );
 
         if (Array.isArray(scenarioOptions)) {
-            return (
-                <Select
-                    onValueChange={this.handleUpdateScenario}
-                    defaultSelection={[scenarioOptions[0].displayName]}
-                    selectedItems={[
-                        scenarioOptions[this.state.selectedScenarioIndex].displayName,
-                    ]}
-                >
-                    {this.renderScenarioOptions(scenarioOptions)}
-                </Select>
+            return renderScenarioFastSelect(
```

Comment:
Can we rename this function to `renderScenarioSelect`?

Reply:
Done